style-guide.md

```diff
@@ -21,51 +21,51 @@ export default () => (
 ✅ DO:
 
 ```js
-// Ejemplo
+// Exemple
 import React from "react"
 export default () => (
-  <div style={{ color: `purple`, fontSize: `72px` }}>Hello Gatsby!</div>
+  <div style={{ color: `purple`, fontSize: `72px` }}>Salut Gatsby!</div>
 )
 ```
 
-✅ ALSO OKAY:
+✅ OK:
 
 ```js
-// Ejemplo
+// Exemple
 import React from "react"
 export default () => (
-  <div style={{ color: `purple`, fontSize: `72px` }}>¡Hola Gatsby!</div>
+  <div style={{ color: `purple`, fontSize: `72px` }}>Salut Gatsby!</div>
 )
 ```
 
 ❌ DON'T:
 
 ```js
-// Ejemplo
+// Exemple
 import React from "react"
 export default () => (
   // 'purple' is a CSS keyword
-  <div style={{ color: `morado`, fontSize: `72px` }}>¡Hola Gatsby!</div>
+  <div style={{ color: `morado`, fontSize: `72px` }}>Salut Gatsby!</div>
 )
 ```
 
 ❌ DEFINITELY DON'T:
 
 ```js
-importar Reaccionar desde "reaccionar"
-exportar defecto () => (
-   <div estilo = {{color: `morado`, fontSize:` 72px`}}> ¡Hola Gatsby! </div>
+import React from "react"
+export default () => (
+   <div style = {{color: `morado`, fontSize:` 72px`}}> Salut Gatsby! </div>
```

Comment:
Great translation. Just some code styling : 

```suggestion
   <div style={{color: `morado`, fontSize:` 72px`}}>Salut Gatsby !</div>
```

Reply:
That is the Definitely not section and in the [translation guide](https://www.gatsbyjs.org/contributing/gatsby-docs-translation-guide/#text-in-code-blocks) it had definitely don't in spanish rather than english

Reply:
Ok I get it. You're right! I resolve the bad RFCs.

Reply:
What about using this?

```
importer React de "react"
exporter par défaut () => (
   <div style = {{color: `mauve`, fontSize:` 72px`}}>Salut Gatsby !</div>
)
```

Reply:
Go for it ! 👍